src/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+} from 'react-router-dom';
+import './styles/App.css';
+import Main from './pages/Main';
+
+const App = () => (
```

Comment:
## Good job adding the ability to add and delete doctors 👍

_However, there are some issues within the implementation:_

- [ ] When I add a doctor, some data won't be recorded in the DB, no matter how many times I try:

![image](https://user-images.githubusercontent.com/57421823/225495816-648df757-99e5-4940-954a-03babaabad70.png)

This is the transaction to the database:

![image](https://user-images.githubusercontent.com/57421823/225496294-979d2f24-14d2-46b1-84ee-e48293b79bdf.png)

- [ ] When I create doctors, their portraits are getting blocked and won't show up:

### Network tab:

![image](https://user-images.githubusercontent.com/57421823/225496607-11c5b6de-86b0-47e2-a39c-4d7b6dfe3baa.png)

### Console:

![image](https://user-images.githubusercontent.com/57421823/225497100-b48051a3-84dc-4930-b4b0-16df9de04ba8.png)


- [ ] When I try to make a reservation, the new doctors I created won't show up for me in the list:

![image](https://user-images.githubusercontent.com/57421823/225496994-b5ca168d-8ec7-4789-bbbb-3da80c511e11.png)

Therefore, I can't reserve with my new doctors.

Please, check if these issues are working for you.  If these are bugs, kindly fix them so your app works as intended.  Deleting doctors is working well.  💪 

_note:  All issues were tested both on Chrome and Firefox._

Reply:
@Meltrust I am so sorry. I did not notice that some datas were not uploading. I will work on them asap